deepmd/main.py

```diff
@@ -255,6 +255,11 @@ def main_parser() -> argparse.ArgumentParser:
         default=None,
         help="Finetune the frozen pretrained model.",
     )
+    parser_train.add_argument(
+        "--use-pretrain-script",
+        action="store_true",
+        help="Use model parameters from the script of the pretrained model instead of user input when doing finetuning. Note: This behavior is default and unchangeable in TensorFlow.",
```

Comment:
Do we plan to change it in TF? If so, we may have an issue to track it.

Reply:
I've opened an issue about this https://github.com/deepmodeling/deepmd-kit/issues/3869. I think we may change it in the future if there is a clear demand.